torchci/lib/jobUtils.ts

```diff
@@ -95,3 +106,133 @@ export function removeJobNameSuffix(
 
   return jobName.replace(REMOVE_JOB_NAME_SUFFIX_REGEX, replaceWith);
 }
+
+export function isSameFailure(
+  jobA: RecentWorkflowsData,
+  jobB: RecentWorkflowsData
+): boolean {
+  if (
+    jobA.name === undefined ||
+    jobA.name === "" ||
+    jobB.name === undefined ||
+    jobB.name === ""
+  ) {
+    return false;
+  }
+
+  // Return true if two jobs have the same failures. This is used to figure out
+  // broken trunk and other similar failures
+  const jobANameNoSuffix = removeJobNameSuffix(jobA.name);
+  const jobBNameNoSuffix = removeJobNameSuffix(jobB.name);
+
+  if (
+    !jobANameNoSuffix.includes(jobBNameNoSuffix) &&
+    !jobBNameNoSuffix.includes(jobANameNoSuffix)
```

Comment:
why won't a simple equality check work here?

Reply:
Ah yeah, silly me.  An equality check should do the same thing